GameEngine/Services/Helper.cs

```diff
@@ -0,0 +1,38 @@
+﻿using GameEngine.Entities;
+
+namespace GameEngine.Services
+{
+    /// <summary>
+    /// The class contains common methods.
+    /// </summary>
+    public class Helper
+    {
+        private static Random _random = new Random();
+
+        /// <summary>
+        /// Generates a random number in the interval.
+        /// </summary>
+        /// <param name="minLimit">Min limit.</param>
+        /// <param name="maxLimit">Max limit.</param>
+        /// <returns>Random number.</returns>
+        public static int GenerateRandomCoordinates(int minLimit, int maxLimit)
+        {
+            return _random.Next(minLimit, maxLimit);
```

Comment:
This method has no added value. Use:
_random.Next(minLimit, maxLimit);
where method is used.

Reply:
Done.